docs/source/events/examples/payment-status-change.yaml

```diff
@@ -17,5 +17,17 @@ currency_code: EUR
 # Payment method specific data (stripe intent ID in this case)
 payment_data:
   transaction_id: pi_3NOKIODvbS6ezf4w21zIR6Ft
+# Line items:
+line_items:
+- name: one-off
+  amount: 10
+  quantity: 9
+  tax_rate: 0
+  recurrence_interval: null
+- name: monthly-item
+  amount: 10
+  quantity: 1
+  tax_rate: 0
+  recurrence_interval: P1M
```

Comment:
What can the recurrence intervals be? What does P1M mean? What else could it be?

Reply:
Again this is an example embedded in the main article about this type of event (`payment-status-change.rst`).